cmake/projects/Lager/hunter.cmake

```diff
@@ -0,0 +1,33 @@
+# !!! DO NOT PLACE HEADER GUARDS HERE !!!
+
+include(hunter_add_version)
+include(hunter_cacheable)
+include(hunter_cmake_args)
+include(hunter_download)
+include(hunter_pick_scheme)
+
+hunter_add_version(
+    PACKAGE_NAME
+    Lager
+    VERSION
+    0.0.1-dbc1fde
+    URL
+    "https://github.com/Bjoe/lager/archive/68e69c30667c12e01ef98a65e3ecef12c461f2e1.zip"
```

Comment:
It need to have a repo in cpp-pm

Reply:
Done https://github.com/cpp-pm/lager

Reply:
Unfortunately I pushed my `hunter` branch to  https://github.com/cpp-pm/lager repo and then I saw your comment on `cppast` PR and rename this branch. But I cannot delete the `hunter` branch in https://github.com/cpp-pm/lager repo. Can you please delete this branch?

Reply:
Done